week1/velog-clone/index.html

```diff
@@ -7,69 +7,121 @@
+              fill="#ececec"
             ></path>
           </svg>
-        </button>
-        <a class="search__button">
-          <svg width="17" height="17" viewBox="0 0 17 17">
-            <path
-              fill-rule="evenodd"
-              d="M13.66 7.36a6.3 6.3 0 1 1-12.598 0 6.3 6.3
+        </div>
+        <div class="header__contents">
+          <button class="darkmode__button">
```

Comment:
[bem방법론](https://velog.io/@fromzoo/css-%EB%B0%A9%EB%B2%95%EB%A1%A0-BEM-%EB%B0%A9%EC%8B%9D)에 따라 바꿔보면
```suggestion
    <div class="header__contents">
          <button class="header__button-darkmode">
```
키-밸류(key-value) 타입은  성질-내용이라서 순서를 바꿔봐따!